### ISSUE TITLE
focus lost on cards

### ISSUE THREAD

**Issue by stef-barburou** [DIS-SIN/Busrides#142]
When I use tab to navigate through the main Busrides pages, when I tab beyond the subtitle (ie. Going Remote Guide), I cannot visually see what is selected next. I suspect it's the title of each card, but it is not visually apparent and does not put a border around anything. To investigate.

**Comment by stef-barburou** [comment]
it'S actually the next card , but this is being referenced in another major card ( #149 ) suggesting we close this one.
